taigit/src/libraries/Taiga.ts

```diff
@@ -78,4 +78,124 @@ task_history(taskId : number) : Promise<Object> {
+    }
+    return (total_task_details);
+}
+
+/**
+ * @summary Get the Task count with its status for a sprint in  a project
+ * @param sprint_id : Milestone Id 
+ * @param project_id: Project Id 
+ * @param sprint_name: Sprint name
+ * @returns array of each members along with task count for a single sprint 
+ * {
+ *      inprogress_task_count: number
+ *      name: string
+ *      new_task_count: number
+ *      sprint_name: string
+ *      task_ready_for_test_count: number
+ *      task_with_unknown_status_count: number
+ *      total_task_count: number 
+ * }
+ */
+
+async function
+get_task_details(sprint_id: number, project_id: any, sprint_name: string)  : Promise<Object> {
+
+    let dict_task_count: any = {};
+    let json_obj: any = {};
+    let big_obj: any = [];
+    let response = (await axios.get(`https://api.taiga.io/api/v1/tasks?milestone=${sprint_id}&&project=${project_id}`)).data;
+    
+    for (let task_name of response) {
+        let name : string;
+        if (task_name.assigned_to_extra_info != null) {
+            name = task_name.assigned_to_extra_info.full_name_display;
+        } else {
+            name = task_name.owner_extra_info.full_name_display;
+        }
+        let task_status_name = task_name.status_extra_info.name;
+        let task_status = task_name.status_extra_info.is_closed;
+        let nameKey: any;
+        if (name in dict_task_count) {
+            nameKey = dict_task_count[name];
+            dict_task_count[name] = nameKey + 1;
+
+            for (let i = 0; i < big_obj.length; i++) {
+                if (name === big_obj[i].name) {
+                    big_obj[i].total_task_count = big_obj[i].total_task_count + 1;
+                    if (task_status === true) {
+                        big_obj[i].closed_task_count = big_obj[i].closed_task_count + 1;
+                    } else {
+                        if (task_status_name === "Ready for test") {
+                            big_obj[i].task_ready_for_test_count = big_obj[i].task_ready_for_test_count + 1;
+                        } else if (task_status_name === "In progress") {
+                            big_obj[i].inprogress_task_count = big_obj[i].inprogress_task_count + 1;
+                        } else if (task_status_name === "New") {
+                            big_obj[i].new_task_count = big_obj[i].new_task_count + 1;
+                        }
+                        else {
+                            big_obj[i].task_with_unknown_status_count = big_obj[i].task_with_unknown_status_count + 1;
+                        }
+                    }
+                 }
+            }
+        } else {
+            // This block will execute only once for 
+            // for each team member
+            dict_task_count[name] = 1;
+            json_obj = {};
+            json_obj['name'] = name;
+            json_obj['sprint_name'] = sprint_name;
+            json_obj['total_task_count'] = 1;
```

Comment:
Should we be assuming that a team member will always have a total task count of 1 (presuming they've not done work)?

Reply:
I believe this may cause some changes in lines 184-195 depending on the answer to this

Reply:
Yea! You are right, I changed it.